components/image/graphviz/Makefile

```diff
@@ -120,25 +134,17 @@ REQUIRED_PACKAGES += library/gd
 REQUIRED_PACKAGES += library/glib2
 REQUIRED_PACKAGES += library/libpoppler
 REQUIRED_PACKAGES += library/libtool/libltdl
+REQUIRED_PACKAGES += library/libwebp
 REQUIRED_PACKAGES += library/math/gts
 REQUIRED_PACKAGES += library/zlib
 REQUIRED_PACKAGES += print/filter/ghostscript
 REQUIRED_PACKAGES += runtime/lua
-REQUIRED_PACKAGES += runtime/perl-522
-REQUIRED_PACKAGES += runtime/ruby-23
-REQUIRED_PACKAGES += runtime/tk-8
+REQUIRED_PACKAGES += runtime/perl-534
+REQUIRED_PACKAGES += runtime/ruby-26
+REQUIRED_PACKAGES += shell/ksh93
 REQUIRED_PACKAGES += system/library
```

Comment:
Probably also tcl and tk are build dependencies, aren't they?

Reply:
Yes, tcl-8 was missing.  Added.  Thank you for the hint.

For tk I'm not sure.  I didn't find anything in the build log suggesting tk is really used during the build, but maybe I just didn't looked hard enough.